examples/active.ini

```diff
@@ -1,10 +1,11 @@
 [acq_params]
-amplitude = 1.0
-p90_us = 4.8
+amplitude = 0.575
+p90_us = 1.7
 deadtime_us = 10.0
 tau_us = 3500.0
 deblank_us = 1.0
 acq_time_ms = 1024.0
+sw_khz = 3.8999532005615936
```

Comment:
sw param not included before, do I want to keep this as added?

Reply:
this is OK, actually

Reply:
b/c you can rationalize that you really want that as a default parameter